mlrun/model_monitoring/stores/kv_model_endpoint_store.py

```diff
@@ -0,0 +1,501 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import typing
+
+import v3io.dataplane
+import v3io_frames
+
+import mlrun
+import mlrun.api.schemas
```

Comment:
we don't want to support client side to use server side objects. we do support the opposite ( server side uses client side ) although we would prefer just having it in a common module where the code inside doesn't rely on any side and is agnostic

Reply:
The thing is that we have moved the `stores` package to the client side because of reusing the sql code in both client and server sides but I see the problem of using the server side objects. All of these issues were related to the fact that until this PR everything in the monitoring flow was based only on the server side object `mlrun.api.schemas.model_endpoints.ModelEndpoint`, even when the user called the `get_model_endpoint()` or `list_model_endpoint()` through the API. 
Let's discuss about it offline.